src/App.tsx

```diff
@@ -1,27 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  return (
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+      <button
+        type="button"
+        data-cy="all-button"
+        onClick={() => {
+          getAll()
+            .then(todosFromServer => {
+              setGoods(todosFromServer);
+            });
+        }}
+      >
+        Load all goods
+      </button>
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+      <button
+        type="button"
+        data-cy="first-five-button"
+        onClick={() => {
+          get5First()
+            .then(todosFromServer => {
+              setGoods(todosFromServer);
+            });
+        }}
```

Comment:
Same here and below. Please fix all cases. Also, don't forget to handle possible errors
![image](https://user-images.githubusercontent.com/96409563/229450242-43b04337-1b09-4b3b-aec7-d07e753bfb0c.png)